homepage/repository.md

```diff
@@ -18,7 +18,7 @@ are labeled explicitly.
 You can add the repository with a simple command:
 
 {% clipboard %}
-    helm repo add eclipse-iot https://eclipse.org/packages/charts
+    helm repo add eclipse-iot <https://eclipse.org/packages/charts>
```

Comment:
Hm, does the `<...>` mean that this one is optional?
Or will this command succeed with this syntax?

Reply:
My markdown linter mentioned this (see https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md034---bare-url-used)

If this might break something on the website, I'll revert it.

Reply:
As we don't need this to be a clickable link, I would remove the `<..>` again ..